.editorconfig

```diff
@@ -6,11 +6,3 @@ root = true
 charset = utf-8
 indent_style = space
 indent_size = 4
-
-[*.java]
-indent_style = space
-indent_size = 4
-
-[*.xml,*.xslt]
-indent_style = space
-indent_size = 4
```

Comment:
I am not familiar with `.editorconfig` but why is it removed?

Reply:
Same config as the first config which include all files.